semantic/src/Semantic/Api/Terms.hs

```diff
@@ -69,74 +71,95 @@ showTermParsers = preciseParsers
 class ShowTerm term where
   showTerm :: (Has (Reader Config) sig m) => term Loc -> m Builder
 
-instance (ShowTermBy term) => ShowTerm term where
-  showTerm = showTermBy @term
```

Comment:
We used to have two typeclasses per serialization interface, due to alacarte/precise syntax differences. Due to type weirdness, we still need one, but I removed the other `TermBy` classes.